### PR TITLE
Deprecate podspec

### PR DESCRIPTION
**Pull request by x2on**
Mark `GoogleAnalytics-iOS-SDK` deprecated as Google now provide their own Specs.

### CODE REVIEW

File: Specs/GoogleAnalytics-iOS-SDK/3.12/GoogleAnalytics-iOS-SDK.podspec.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
     "ios": "5.0"
   },
   "requires_arc": true,
+  "deprecated": true,
+  "deprecated_in_favor_of": "GoogleAnalytics",
```
Comment (keith) [inline Specs/GoogleAnalytics-iOS-SDK/3.12/GoogleAnalytics-iOS-SDK.podspec.json]: No need for both of these, you can just add `deprecated_in_favor_of`

**Comment by x2on** [comment]
@keith good to know - i changed it.

**Comment by allaire** [comment]
@x2on @keith can someone of you explain the difference between the Google/Analytics pod and the GoogleAnalytics one? It's pretty unclear, except that the Google/Analytics seems less heavy?

Thanks

**Comment by beriberikix** [comment]
Hi Mathieu,

I work on mobile SDKs at Google and can help. The Google/Analytics Pod is part of our new improved workflow for [Google APIs on iOS](https://developers.google.com/ios) that we launched earlier this year. It includes a simpler flow to get API credentials and a simpler, config-based API to bootstrap those services. The intention is that new developers should start using Google/Analytics and everyone else should follow when they're ready. You can check out the Google Analytics-specific guide on our [developer site](https://developers.google.com/analytics/devguides/collection/ios/v3/start).

Sorry for the confusion and lack of clarity - we plan on putting more explanation in the README to make that clear.

HTH!

**Comment by allaire** [comment]
Hi @beriberikix !

Thanks for shedding some lights, it's all make more sense now!

Two more questions: 

Will tag manager be a part of the new Google APIs? I don't see it anywhere. I see its included in google analytics services when you download it manually though. 

The lib files in the Google/analytics seems much lighter (couple of Ko) compared to the lib in GoogleAnalytics, which is a couple of mo. And the manually downloaded google analytics sdk makes almost 12mo. What so different between these different ways of installing the sdk?

Thanks!

**Comment by beriberikix** [comment]
We will be adding more services to the new flow over time and we are trying to prioritize. That said, once you get the GoogleTagManager keys the current standard way, GTM & GA continue to work together. And may be a minor thing but in the process of cleanup we made the GoogleTagManager an [officially-supported](http://cocoapods.org/owners/6563) Pod and push updates there.

And on size - we can thank dependency management ;) Through CocoaPods we can split out features like [IDFA](http://cocoapods.org/pods/GoogleIDFASupport) and shared internal dependancies.

**Comment by allaire** [comment]
@beriberikix Thanks for the follow up, much appreciated!

**Comment by beriberikix** [comment]
:+1:

**Pull request merged**